### PR TITLE
(actions) Add a github action ES-Lint

### PR DESCRIPTION
**Pull request by AVRahul**
Signed-off-by: AVRahul <a.v.rahul11@gmail.com>

Added a GitHub action for lint check (ES-Lint) with its local eslintrc config file for the branch `revamp-3.0`

### CODE REVIEW

File: website/.eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+**/node_modules/
```
Comment (IsAmrish) [inline website/.eslintignore:1]: can we also add `build` directory?

File: .github/workflows/eslint.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+name: Lint
```
Comment (satyapriyamishra222) [inline .github/workflows/eslint.yml:1]: TSLint

Comment (AVRahul) [reply .github/workflows/eslint.yml:1]: ok, will go ESLint

**Review by IsAmrish — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]

**Review by rakeshPRaghu — APPROVED** [review]
LGTM

**Review by Pallavi-PH — APPROVED** [review]

**Pull request merged**